composer.json

```diff
@@ -31,6 +31,7 @@
     "require-dev": {
         "symfony/phpunit-bridge": "^3.4.19|^4.1.8|^5.0",
         "symfony/debug": "^2.7",
```

Comment:
looks like a very outdated version is used

Reply:
fixed